### PR TITLE
added css

### PR DESCRIPTION
**Pull request by simrat12**
Adding CSS for pop up box

### CODE REVIEW

File: api/src/frontend/pages/freelancer/profile.tsx [hunk 1]
```diff
@@ -78,38 +78,71 @@ type MessageFormProps = {
...
-                    id="body"
-                    value={body}
-                    onChange={(event) => setBody(event.target.value)}
-                    style={{ marginBottom: '20px' }} 
-                />
-                <button type="submit" value="submit" style={{ marginTop: '10px' }}>Submit</button> 
-            </form> 
-        </div>
+      <div className="message-form-container" style={{ 
+        display: "flex",
```
Comment (samelamin) [inline api/src/frontend/pages/freelancer/profile.tsx]: This CSS should go on the profile stylesheet
We shouldnt have inline CSS

Comment (simrat12) [reply api/src/frontend/pages/freelancer/profile.tsx]: Added CSS to stylesheet

**Review by samelamin — APPROVED** [review]

**Review by cuteolaf — APPROVED** [review]

**Review by f-gate — APPROVED** [review]
looks good, anyone know if we should be using dynamic sizes as opposed to px?

**Comment by samelamin** [comment]
> looks good, anyone know if we should be using dynamic sizes as opposed to px?
I prefer dynamic sizes, seems to be better on mobile

**Pull request merged**